src/App.js

```diff
@@ -36,14 +37,15 @@ import 'react-datepicker/dist/react-datepicker.css';
 class App extends Component {
   constructor(props){
     super(props)
+    // set this to <false> if you don't want to have to read from an encoded URL, <true> instead
+    const readFromEncodedUrl = false;
     this.state = {
       numberOfDays: 0,
       result: "",
       resultDays: 0,
-      calculatorInfo: this.decodeToState(),
+      calculatorInfo: readFromEncodedUrl ? this.decodeToState() : excludedDates,
```

Comment:
this is awesome, thank you for doing this!

Reply:
I am glad to help